fenix/app/src/androidTest/java/org/mozilla/fenix/ui/OnboardingTest.kt

```diff
@@ -64,11 +64,7 @@ class OnboardingTest {
             )
             verifyToolbarPlacementCard(isBottomChecked = true, isTopChecked = false)
             verifySignInToSyncCard()
-            verifyPrivacyProtectionCard(
-                settings = activityTestRule.activity.getSettings(),
-                isStandardChecked = true,
-                isStrictChecked = false,
-            )
+            verifyPrivacyProtectionCard(isStandardChecked = true, isStrictChecked = false)
```

Comment:
This change regarding `verifyPrivacyProtectionCard` (here and below) is not needed.